xtask/src/macros.rs

```diff
@@ -1,3 +1,4 @@
+#[macro_export]
```

Comment:
issue: Do not use macro_export for internal macros, as it means for exporting the macro outside the crate. Instead, you can `pub(crate) use` it to export it from the module: https://play.rust-lang.org/?version=stable&mode=debug&edition=2024&gist=93ee277d14ed04bb40eda0b502961233

Reply:
Done